src/image-question/components/runtime.tsx

```diff
@@ -1,33 +1,19 @@
-import React, { useMemo, useState } from "react";
+import React from "react";
 import { IRuntimeQuestionComponentProps } from "../../shared/components/base-question-app";
-import { renderHTML } from "../../shared/utilities/render-html";
 import { IAuthoredState, IInteractiveState } from "./app";
 import { Runtime as DrawingToolRuntime } from "../../drawing-tool/components/runtime";
-import { IAuthoredState as IDrawingAuthoredState } from "../../drawing-tool/components/app";
-import { showModal, getInteractiveSnapshot } from "@concord-consortium/lara-interactive-api";
+import { showModal } from "@concord-consortium/lara-interactive-api";
 import { v4 as uuidv4 } from "uuid";
 import ZoomIcon from "../../shared/icons/zoom.svg";
-import CameraIcon from "../../shared/icons/camera.svg";
 import css from "./runtime.scss";
-import cssHelpers from "../../shared/styles/helpers.scss";
 
-interface IProps extends IRuntimeQuestionComponentProps<IAuthoredState, IInteractiveState> {
-}
+interface IProps extends IRuntimeQuestionComponentProps<IAuthoredState, IInteractiveState> {}
 
 const kGlobalDefaultAnswer = "Please type your answer here.";
 
 export const Runtime: React.FC<IProps> = ({ authoredState, interactiveState, setInteractiveState, report }) => {
-  const { version, imageFit, imagePosition, required, stampCollections, modalSupported } = authoredState;
+  const { required, modalSupported } = authoredState;
```

Comment:
FYI: there will be a conflict here with #58 which removes the `modalSupported` property from authoredState.

Reply:
Right, I need to check how this works, for now I'm ignoring the modal version and I'm focusing on features that work inline.